ui/packages/catalog-ui-search/src/main/webapp/js/widgets/openlayers.polygon.js

```diff
@@ -49,6 +49,19 @@ const translateFromOpenlayersCoordinates = coords => {
     .flatten()
 }
 
+/**
+ * Returns buffer width in meters with the lowest value of 1 meter.
+ */
+const getConstrainedBufferWidth = model => {
+  const bufferWidth =
+    DistanceUtils.getDistanceInMeters(
+      model.get('polygonBufferWidth'),
+      model.get('polygonBufferUnits')
+    ) || 1
```

Comment:
this doesn't prohibit not using a buffer does it?

Reply:
No, this only makes sure we do not apply buffers less than 1 meter